runner/utils/driver/istio/istio.go

```diff
@@ -14,11 +14,21 @@ import (
 	"github.com/spaceuptech/space-cloud/runner/utils/auth"
 )
 
+// Deployment stores the deploymentID
+type Deployment map[string]Replicas
```

Comment:
```suggestion
type Deployments map[string]Status
```
Don't export the types if it is not being used externally